modules/elasticsearch/src/main/java/org/testcontainers/elasticsearch/ElasticsearchContainer.java

```diff
@@ -39,6 +39,8 @@
      */
     @Deprecated
     protected static final String DEFAULT_TAG = "7.9.2";
+    private boolean isOss = false;
```

Comment:
I suggest to use more descriptive boolean variable name - its hard (at least for me) to understand, what exactly `ml` variable means in this code and to understand it its necessary for reader to go deeper in code logic

Reply:
Right. Good proposal.